konf.py

```diff
@@ -0,0 +1,139 @@
+#! /usr/bin/env python3
+
+import yaml
+import jinja2
```

Comment:
This is also going to need to be in `requirements.txt`.

Reply:
Okay, can it be confusing for Jenkins? We agreed with IS for security reason to use the Debian packages there.

Reply:
What if I make this a snap? I mean, the `konf.py` and the Jinja2 `templates` folder, this would also help us with what we talk about having each file in its own repository.

Reply:
As discussed, I don't think we should publish this config generator as a package.

This is because, firstly, we will only ever use it for one thing - for applying k8s configs from our Jenkins installation - so packaging is not any use for general distribution.

Secondy, when we make changes to these configs, we'll likely want to coordinate the release of those changes with a change to one of our projects, and releasing those changes out of sync could potentially be catastrophic.

If we keep the configs in this repo, then our release process looks like:

1. push these changes to master
2. run the Jenkins job which will pull down the latest version from master and run it

But if we were to publish it as a package, then our release flow gets more complicated:

1. push these changes to master
2. publish the package
3. update the version of that package on Jenkins
4. run the Jenkins job

I'm worried that this increases the cost of step getting forgotten, possibly to catastrophic effect, especially when I don't see an obvious benefit to packaging it up.

Thirdly, if we package it up then this repo will get more complicated and harder to understand, as it needs to contain packaging infrastructure.